packages/cli/README.md

```diff
@@ -273,6 +273,30 @@ bubblewrap fingerprint generateAssetLinks <flags>
 Flags:
  - `--output=<filename>`: path from where to load the project configuration.
 
+## `play`
+
+:warning: This is an experimental feature.
+
+Manages the artifacts for your Google Play Project.
+
+Usage:
+
+```
+bubblewrap play --serivceAccountFile="/path/to/serivce/account"  --retain 86 --publish
+```
+
+This tool requires a service account file to work correctly. Please see [this documentation](https://github.com/chromeos/pwa-play-billing#setup-a-service-account) for setting up a service acocunt.
+
+Options:
+  - `--publish`: publishes the prebuilt file to the Google Play Store under the internal track or specificed track.
```

Comment:
I think this would read better as "publishes ... to the specified track (defaults to internal)."

Reply:
Done.